version/version.go

```diff
@@ -17,7 +17,6 @@ var (
 	// P2PProtocol versions all p2p behaviour and msgs.
 	// This includes proposer selection.
 	P2PProtocol uint64 = 8
-
```

Comment:
Maybe leave this out. Not that it matters much, but it's just noise and can cause hassle with cherry-picking (e.g. if some other commit moves this file).

Reply:
Will revert prior to merge. Using this to make sure it all runs.

Reply:
reverted